creator-node/compose/env/commonEnv.sh

```diff
@@ -1,2 +1,7 @@
 export CREATOR_NODE_IS_USER_METADATA=false
-export DEV_MODE=true
\ No newline at end of file
+export DEV_MODE=true
+export delegateOwnerWallet=DELEGATE_OWNER_WALLET
+export delegatePrivateKey=DELEGATE_PRIVATE_KEY
+export creatorNodeEndpoint=CREATOR_NODE_ENDPOINT
+export spOwnerWallet=SP_OWNER_WALLET
+export COMPOSE_HTTP_TIMEOUT=200
```

Comment:
this is upped to allow more breathing room for health checks to pass when they are competing to run in parallel provisioning.